docs/building_the_project_with_dpcpp.rst

```diff
@@ -0,0 +1,446 @@
+.. _building_the_project_with_dpcpp:
+
+Building the Project with DPC++
+===============================
+
+This page describes building the oneMKL interface library with either the Intel(R) oneAPI DPC++ Compiler or open-source oneAPI DPC++ Compiler.
+For guidance on building the project with AdaptiveCpp, see :ref:`building_the_project_with_adaptivecpp`.
+
+.. _build_setup_with_dpcpp:
+
+Build Setup
+###########
+
+#. 
+   Install Intel(R) oneAPI DPC++ Compiler (select the variant as described in
+   :ref:`Selecting a Compiler<selecting_a_compiler>`).
+
+#. 
+   Clone this project to ``<path to onemkl>``\ , where ``<path to onemkl>``
+   is the root directory of this repository.
+
+#. 
+   Build and install all `required dependencies <https://github.com/oneapi-src/oneMKL?tab=readme-ov-file#software-requirements>`_. 
+
+.. _build_introduction_with_dpcpp:
+
+Introduction
```

Comment:
I wouldn't expect an "Introduction" section in a "how to build" page, maybe "Build Commands"?